modules/efk-logging-curator-tolerations.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-curator.adoc
+
+[id="efk-logging-curator-tolerations_{context}"]
+= Using tolerations to control the Curator pod placement
+
+You can ensure which node the Curator pod runs on
```

Comment:
it looks not very clear. Something like "You can control on which node the Curator pod runs using tolerations on the pod" sounds clear.